react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,26 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+const returnOneArray = (array) => array.reduce((acc, val) => acc.concat(val), []);
+
+function min(...params) {
+  if (params.length === 0) return undefined;
+
+  return isArray(...params)
+    ? Math.min(...returnOneArray(params))
```

Comment:
You wouldn't need to call this function to return one array if you use spread properly. :)